CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+## Welcome!
+
+Hello! It's awesome that you want to contribute to the Discord bridge, by doing
+so you are helping our users and the wider Matrix ecosystem. This document
+lists the requirements for your work to be accepted into our repository.
+
+For clarity reasons, work can either be a pull request or an issue. We find
+both quite valuable to the project. One last thing before we begin is that you
```

Comment:
'One last thing before we begin' seems a bit unnecessary, since it is only the second real point and there is a whole paragraph afterwards. Also, license is misspelled as 'licence' each time it appears. Could be rephrased as:
`We find both quite valuable to the project. Please note that your work must abide by the Apache 2 license found in the LICENSE file.`